database/redis/build.gradle

```diff
@@ -0,0 +1,45 @@
+plugins {
+    id 'java'
+    id 'idea'
+    id 'io.quarkus'
```

Comment:
This is not required here. At least redis shouldn't depend on quarkus. Those are two different technologies.

Reply:
done